.circleci/config.yml

```diff
@@ -60,12 +60,20 @@ jobs:
       - run:
           name: e2e test
           command: |
+            set +e
             if [[ $CIRCLE_BRANCH != *"master"* || ! -z $CIRCLE_TAG ]] ; then
               E2EPROFILE=saucelight npm run test:e2e
             else
               npm run test:e2e
             fi
-  
+            STATUS=$?;
```

Comment:
i prefer this way because I usually try to keep one assignment per line.

Reply:
Ok, then you can use this now. :)